SofaKernel/modules/SofaSimulationCore/src/sofa/simulation/BaseMechanicalVisitor.h

```diff
@@ -240,6 +240,10 @@ class SOFA_SIMULATION_CORE_API BaseMechanicalVisitor : public Visitor
     sofa::helper::vector< sofa::core::ConstMultiVecId > readVector;
     sofa::helper::vector< sofa::core::MultiVecId > writeVector;
 #endif
+
+private:
+    static const std::string fwdVisitorType;
```

Comment:
it wont change anything I suppose but constexpr may be better in this case ? (and you also inline initialize the value here)

Reply:
constexpr is not allowed on strings. It is allowed from C++20

Reply:
right, I forgot that.